integration-tests/src/test/groovy/com/okta/sdk/tests/it/util/ITSupport.groovy

```diff
@@ -65,6 +71,32 @@ abstract class ITSupport implements ClientProvider {
         }
     }
 
+    /**
+     * Some Integration test operations exhibit flakiness at times due to replication lag at the core backend.
+     * We therefore add delays between some test operations to ensure we give sufficient
+     * interval between invoking operations on the backend resource.
+     *
+     * This delay could be specified by the following in order of precedence:
+     * - System property 'okta.it.operationDelay'
+     * - Env variable 'OKTA_IT_OPERATION_DELAY'
+     */
+    long getTestOperationDelay() {
+        Long testDelay = Long.getLong(IT_OPERATION_DELAY)
+
+        if (testDelay == null) {
+            if (StringUtils.isNotBlank(System.getenv("OKTA_IT_OPERATION_DELAY"))) {
+                try {
+                    testDelay = Long.valueOf(System.getenv("OKTA_IT_OPERATION_DELAY"))
```

Comment:
nit, you could combine these two lines with something like:

```java
testDelay = Long.valueOf(System.getenv().getOrDefault("OKTA_IT_OPERATION_DELAY", "0"))
```

Reply:
done.